python_ta/debug/accumulation_table.py

```diff
@@ -101,14 +104,15 @@ def _create_iteration_dict(self) -> dict:
         and loop variable to its respective value during each iteration
         """
         return {
-            "iteration": list(range(len(list(self.loop_variables.values())[0]))),
+            "iteration": list(range(len(list(self.loop_accumulators.values())[0]))),
```

Comment:
This is interesting. You should not assume that `self.loop_accumulators` is non-empty. A better precondition is that *at least one of* `self.loop_variables` and `self.loop_accumulators` is non-empty.

Somewhere in `setup_table` you can add an explicit check for this with an `assert` to raise an error if both are empty.